agenthub/light_codeact_agent/README.md

```diff
@@ -0,0 +1,20 @@
+# LightCodeAct: A Simplified CodeAct Agent Framework
+
+LightCodeAct is a simplified version of the CodeAct agent ([paper](https://arxiv.org/abs/2402.01030), [tweet](https://twitter.com/xingyaow_/status/1754556835703751087)). The original CodeAct idea consolidates LLM agents' **act**ions into a unified **code** action space for both *simplicity* and *performance*. LightCodeAct aims to make the agent more accessible and easier to work with for a wider range of language models, while still maintaining the core principles of the CodeAct framework.
+
```

Comment:
> LightCodeAct aims to make the agent more accessible and easier to work with for a wider range of language models

Could you plz elaborate more on this point? For example which LLM codeact not support but lightcodeact can work better.

Reply:
I needed to put it in draft mode; my bad. In general, it is an attempt for **phi3:14b**, but it still needs a bit of change and is more of a draft at the moment.